mysql/tox.ini

```diff
@@ -20,7 +20,7 @@ deps =
     -rrequirements-dev.txt
 commands =
     pip install --require-hashes -r requirements.txt
-    pytest -v
+    pytest -v -r apP --log-level info
```

Comment:
do we want to keep passing all these options os is it an oversight?

Reply:
No it's an oversight